README.md

```diff
@@ -30,7 +38,7 @@ app_keys = {
 
 access_keys = {
   'jonathankoren' : {	    # a specific account. You can change this, but
```

Comment:
You've looked at this code more recently than I have, so I trust you. ;) I'll rewrite the loop.